README.md

```diff
@@ -73,24 +63,21 @@ See https://ipfs.github.io/js-ipfs-bitswap
 ```sh
 » tree src
 src
-├── components
-│   ├── decision
-│   │   ├── engine.js
-│   │   ├── index.js
-│   │   └── ledger.js
-│   ├── network             # Handles peerSet and open new conns
-│   │   └── index.js
-│   └── want-manager        # Keeps track of all blocks the peer wants (not the others which it is connected)
-│       ├── index.js
-│       └── msg-queue.js    # Messages to send queue, one per peer
 ├── constants.js
+├── decision-engine
+│   ├── index.js
+│   └── ledger.js
 ├── index.js
-└── types
-    ├── message             # (Type) message that is put in the wire
+├── network.js             # Handles peerSet and open new conns
+├─── want-manager          # Keeps track of all blocks the peer (self) wants
+│   ├── index.js
+│   └── msg-queue.js       # Messages to send queue, one per peer
+└─── types
+    ├── message            # (Type) message that is put in the wire
     │   ├── entry.js
     │   ├── index.js
     │   └── message.proto.js
-    └── wantlist            # (Type) track wanted blocks
+    └── wantlist           # (Type) track wanted blocks
         ├── entry.js
         └── index.js
 ```
```

Comment:
A though. I've been the one documenting pretty much the whole Bitswap. I do like to document things and make them easier for other people to use and contribute (its like teaching), however, it is kind of hard to when I'm mostly the one pushing the docs, graphs and writing other ideas down. It would be very helpful when the way that things are designed change (or created), to get the contributor to document as well.

This tree is missing the new notifications.js and I'm sure a lot of other thoughts that went into this. Could you try to capture all of them in writing to the best of your ability so that future selfs or contributors don't have to hit a wall? Thank you :)

Reply:
@dignifiedquire never received an acknowledgment here. Could you share your feedback?